src/foundation.cs

```diff
@@ -6777,11 +6777,17 @@ interface NSStream {
 		[Protocolize]
 		NSStreamDelegate Delegate { get; set; }
 
-		[Export ("propertyForKey:"), Internal]
-		NSObject PropertyForKey (NSString key);
+#if XAMCORE_4_0
+		[Abstract]
```

Comment:
This is wrong, because this means XI can't instantiate NSInputStream objects for Objective-C subclasses.

For instance: NSUrlRequest.BodyStream is an `NSInputStream`, but when fetching that property, XI wouldn't be able to create an instance, because `NSInputStream` is abstract (and it's likely we haven't bound the actual Objective-C subclass, because it can be Apple internal).